src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -107,24 +107,34 @@ export class RecommendationHandler {
         isFirstPaginationCall: boolean,
         promise: Promise<any>
     ): Promise<any> {
-        const timeoutMessage = isCloud9() ? `Generate recommendation timeout.` : `List recommendation timeout`
-        if (isManualTriggerOn && triggerType === 'OnDemand' && (isCloud9() || isFirstPaginationCall)) {
-            return vscode.window.withProgress(
-                {
-                    location: vscode.ProgressLocation.Notification,
-                    title: CodeWhispererConstants.pendingResponse,
-                    cancellable: false,
-                },
-                async () => {
-                    return await asyncCallWithTimeout(
-                        promise,
-                        timeoutMessage,
-                        CodeWhispererConstants.promiseTimeoutLimit * 1000
-                    )
-                }
+        const timeoutMessage = hasVendedIamCredentials()
+            ? `Generate recommendation timeout.`
+            : `List recommendation timeout`
+            if (
```

Comment:
indent looks wrong. also unnecessary template strings (backticks instead of single-quotes)

Reply:
updated